src/utilities/session.ts

```diff
@@ -128,32 +128,35 @@ export class Session {
         return false;
       })
       .catch(e => {
-        console.log("Log in to an OpenShift Cluster to use this extension: " + JSON.stringify(e));
+        vscode.window.showInformationMessage("Log in to an OpenShift Cluster to use this extension: " + JSON.stringify(e));
         this.loggedIntoOpenShift = false;
         return false;
       });
 
     // Cancel request after 5 seconds without a response from the getNamespaceList request.
     // This usually implies a connectivity issue with OpenShift, which could take a minute or more
     // before receiving the timeout response.
-    const timeout: Promise<boolean> = new Promise(resolve => {
-      setTimeout(() => {
+    let timeout: NodeJS.Timeout | undefined = undefined;
+    const timeoutPromise: Promise<boolean> = new Promise(resolve => {
+      timeout = setTimeout(() => {
+        vscode.window.showWarningMessage('Connection timed out on "getNamespaceList"... Please check your internet/VPN connection.');
```

Comment:
Since this message is displayed to the end user, can you remove the function name from this message? Also do you think it's useful to the user to know where in the validation that it's failing versus just knowing that there's a connectivity issue? Lastly, you can also remove the term internet/VPN and reword it to something like this:

"Connection timed out...Please validate the connectivity to OpenShift"